packages/contracts-bedrock/test/kontrol/deployment/KontrolDeployment.sol

```diff
@@ -4,74 +4,12 @@ pragma solidity ^0.8.0;
 import { Deploy } from "scripts/deploy/Deploy.s.sol";
 
 contract KontrolDeployment is Deploy {
```

Comment:
Do we need this file long term? Seems like we could just use `Deploy --sig 'runWithStateDiff()` ? We will remove non fault proof code at some point

Reply:
Bumping this @tynes. I'd instead leave it for another PR. Let's get the execution improvements in, then remove these files and adapt the `make-summary-deployment.sh` script later if that's alright.